mdata/notifierKafka/cfg.go

```diff
@@ -110,4 +111,16 @@ func ConfigProcess(instance string) {
 			log.Fatal(4, "kafka-cluster: configured partitions not in list of available partitions. missing %v", missing)
 		}
 	}
+
+	// get the "newest" offset for all partitions.
+	// when booting up, we will delay consuming metrics until we have
+	// caught up to these offsets.
+	bootTimeOffsets = make(map[int32]int64)
+	for _, part := range partitions {
+		offset, err := client.GetOffset(topic, part, -1)
+		if err != nil {
+			log.Fatal(4, "kakfa-cluster: failed to get newest offset for %s:%d. %s", topic, part)
```

Comment:
Yes, we want this to kill metrictank.  This code is run when metrictank first starts up, if we cant talk to kafka to get the current state of the topics/partitions then we cant start.